content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,744 @@
+environment variables will be used throughout to simplify the instructions:
+
+Variable | Description
+-------- | -----------
+`CTX_CLUSTER1` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the `CLUSTER1` cluster.
+`CTX_CLUSTER2` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the `CLUSTER2` cluster.
+
+For example:
+
+{{< text bash >}}
+$ export CTX_CLUSTER1=cluster1
+$ export CTX_CLUSTER2=cluster2
+{{< /text >}}
+
+## Configure Trust
+
+A multicluster service mesh deployment requires that you establish trust
+between all clusters in the mesh. Depending on the requirements for your
+system, there may be multiple options available for establishing trust.
+See [certificate management](/docs/tasks/security/cert-management/) for
+detailed descriptions and instructions for all available options.
+Depending on which option you choose, the installation instructions for
+Istio may change slightly.
+
+This guide will assume that you use a common root to generate intermediate
+certificates for each cluster. Follow the [instructions](/docs/tasks/security/cert-management/plugin-ca-cert/)
+to generate and push a ca certificate secrets to both the `CLUSTER1` and `CLUSTER2`
+clusters.
+
+{{< tip >}}
+If you currently have a single cluster with a self-signed CA (as described
+in [Getting Started](/docs/setup/getting-started/)), you need to
+change the CA using one of the methods described in
+[certificate management](/docs/tasks/security/cert-management/). Changing the
+CA typically requires reinstalling Istio. The installation instructions
+below may have to be altered based on your choice of CA.
+{{< /tip >}}
+
+## Install Istio
+
+The steps for installing Istio on multiple clusters depend on your
+requirements for network and control plane topology. This section
+illustrates the options for two clusters. Meshes that span many clusters may
+employ more than one of these options. See
+[deployment models](/docs/ops/deployment/deployment-models) for more
+information.
+
+{{< tabset category-name="install-istio" >}}
+
+{{< tab name="Multi-Primary" category-value="multi-primary" >}}
+
+The steps that follow will install the Istio control plane on both `CLUSTER1` and
+`CLUSTER2`, making each a {{< gloss >}}primary cluster{{< /gloss >}}. Both
+clusters reside on the `NETWORK1` network, meaning there is direct
+connectivity between the pods in both clusters.
+
+Each control plane observes the API Servers in both clusters for endpoints.
+
+Service workloads communicate directly (pod-to-pod) across cluster boundaries.
+
+{{< image width="75%"
+    link="multi-primary.svg"
+    caption="Multiple primary clusters on the same network"
+    >}}
+
+**Configure `CLUSTER1` as a primary**
```

Comment:
Maybe to get around the lint error you can make these dummy headings list items for now:
```suggestion
- **Configure `CLUSTER1` as a primary**
```
We can revisit the layout and improve in a followup PR.

Reply:
Maybe something like this is another option? https://stackoverflow.com/questions/52062672/rmarkdown-add-header-under-tabset